### ISSUE TITLE
Migrate to other alternatives of currently stored Yolantis/uCrop for android

### ISSUE THREAD

**Issue by av153k** [hnvn/flutter_image_cropper#391]
Currently, Yolantis/uCrop is stored at jitpack which has been down for more than 12 hrs as of writing this issue. This has caused builds on almost all significant CI/CD to fail. 

Please consider migrating to a package management service where uCrop is stored even where version is older so that builds don't fail.

**Comment by av153k** [comment]
https://github.com/Yalantis/uCrop/issues/869

**Comment by av153k** [comment]
https://github.com/jitpack/jitpack.io/issues/5337

**Comment by av153k** [comment]
https://github.com/jitpack/jitpack.io/issues/5239

**Comment by EthanIsaac** [comment]
Yep, more than 12 hours dead and no response from them. Could you consider migrating somewhere else?
